src/workflow/default_executor.ts

```diff
@@ -15,18 +25,23 @@ export class DefaultWorkflowExecutor implements WorkflowExecutor {
       setContext: (key, value) => {
         context[key] = value;
       },
+      originalInput: input,
     };
 
-    let finalOutput: any;
+    let state: any = input;
```

Comment:
`curStepOutput` or `curOutput` sounds more appropriate than the generic name `state`, which doesn't offer anything by its face value.

Reply:
Also, it's better not to initialize it with the input value.

Reply:
done